backend/app/resources.py

```diff
@@ -33,25 +37,27 @@ class ItemsList(Resource):
     def get(self):
         return marshal(Item.query.all(), item_marshaller)
 
+    @jwt_required
     def post(self):
         args = creating_item_parser.parse_args()
         item = Repository.create_and_add(Item, args)
         if not item:
-            return 404
-        db.session.commit()
+            return {'message': 'Something went wrong'}, 500
```

Comment:
Aaa, backend should never return 500! And `Something went wrong` is probably the worst message, something like `Something is not okay, but I won't tell you what you need to guess, let's play a game, hehe`